spec/system/casa_org/edit_spec.rb

```diff
@@ -2,6 +2,9 @@
 
 RSpec.describe "casa_org/edit", type: :system do
   let(:organization) { build(:casa_org) }
+  let!(:languages) do
```

Comment:
are these used in the test somewhere? Or does the test fail when they are not present?

Reply:
It was failing. It's to populate the organization's languages list. But It would also pass if I just used the safe operator where the languages list is referenced(casa_org's language partial view). I think it would be neater